### PR TITLE
helm support option --cluster-api-endpoint

### PR DESCRIPTION
**Pull request by my-git9**
/kind bug

**What this PR does / why we need it**:
fix helm chart to make it support option --cluster-api-endpoint in Karmada agent.

**Which issue(s) this PR fixes**:
Fixes https://github.com/karmada-io/karmada/issues/2291

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Fix helm chart to make it support option --cluster-api-endpoint in Karmada agent.
```

### CODE REVIEW

File: charts/karmada/README.md [hunk 1]
```diff
@@ -196,6 +197,7 @@ helm install karmada-scheduler-estimator -n karmada-system ./charts/karmada
...
 | `agent.clusterName`                       | Name of the member cluster                                                                                                                                                                                                                    | `""`                                                                                                                                                       |
+| `agent.clusterEndpoint`                   | Api endpoint of the member cluster                                                                                                                                                                                                            | `""`                                                                                                                                                       |
```
Comment (lonelyCZ) [inline charts/karmada/README.md:200]:
```suggestion
| `agent.clusterEndpoint`                   | Server endpoint of the member cluster                                                                                                                                                                                                            | `""`                                                                                                                                                       |
```

File: charts/karmada/README.md [hunk 2]
```diff
@@ -99,6 +99,7 @@ Edited values.yaml
 installMode: "agent"
 agent:
   clusterName: "member"
+  clusterEndpoint: "https://10.0.0.1:16443"
```
Comment (lonelyCZ) [inline charts/karmada/README.md:102]:
```suggestion
  clusterEndpoint: "https://X.X.X.X:6443"
```

**Review by lonelyCZ — COMMENTED** [review]
Looks good! There are some suggestions, how do you think?

**Comment by Poor12** [comment]
I think clusterEndpoint is not a necessary field. May add if-judge to the chart template.

**Comment by my-git9** [comment]
> Looks good! There are some suggestions, how do you think?

Updated

**Comment by my-git9** [comment]
> I think clusterEndpoint is not a necessary field. May add if-judge to the chart template.

Updated

**Comment by lonelyCZ** [comment]
> Which issue(s) this PR fixes:
> Fixes # https://github.com/karmada-io/karmada/issues/2291

Please update the pr description, that will trace the relative issue.

> Which issue(s) this PR fixes:
> Fixes https://github.com/karmada-io/karmada/issues/2291

**Comment by my-git9** [comment]
> > Which issue(s) this PR fixes:
> > Fixes # #2291
> 
> Please update the pr description, that will trace the relative issue.
> 
> > Which issue(s) this PR fixes:
> > Fixes #2291

Updated

**Review by RainbowMango — APPROVED** [review]
/approve
Thanks.

**Pull request merged**